config/webpack/development.js

```diff
@@ -1,5 +1,5 @@
-process.env.NODE_ENV = process.env.NODE_ENV || 'development'
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
 
-const environment = require('./environment')
+const environment = require('./environment');
```

Comment:
Is there any value is also calling `environment.splitChunks();` in development.js?  Would it make the dynamic recompilation faster by recompiling a smaller chunk?

Reply:
I don't _think_ it has an effect unless the corresponding webpack config is set to compile.